lib/src/postgrest_transform_builder.dart

```diff
@@ -15,9 +15,23 @@ class PostgrestTransformBuilder<T> extends PostgrestBuilder<T> {
   /// Performs horizontal filtering with SELECT.
   ///
   /// ```dart
-  /// postgrest.from('users').select('id, messages');
+  /// postgrest.from('users').insert().select<PostgrestList>('id, messages');
   /// ```
-  PostgrestTransformBuilder select([String columns = '*']) {
+  /// ```dart
+  /// postgrest.from('users').insert().select<PostgrestListResponse>('id, messages', FetchOptions(count: CountOption.exact));
+  /// ```
+  ///
+  /// By setting [FetchOptions.count] to non null or [FetchOptions.forceResponse] to `true` in `upsert`/`update`/`insert`/`delete`, the return type is `PostgrestResponse<T>`.
+  ///
+  /// Allowed types for [R] are:
+  /// - [PostgrestList]
+  /// - [PostgrestMap]
+  /// - [PostgrestMap?]
+  /// - [PostgrestListResponse]
+  /// - [PostgrestMapResponse]
+  /// - [PostgrestResponse]
```

Comment:
I suppose we should update this one as well!
```suggestion
  /// - [List<Map<String, dynamic>>]
  /// - [Map<String, dynamic>]
  /// - [Map<String, dynamic>?]
  /// - [PostgrestResponse<List<Map<String, dynamic>>>]
  /// - [PostgrestResponse<Map<String, dynamic>>]
  /// - [PostgrestResponse<Map<String, dynamic>?>]
```